http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpWsmDao.scala

```diff
@@ -156,6 +156,15 @@ class HttpWsmDao[F[_]](httpClient: Client[F], config: HttpWsmDaoConfig)(implicit
 
       // Step 2: call LZ for LZ resources
       lzResourcesByPurpose <- listLandingZoneResourcesByType(landingZoneId, userToken)
+      region <- lzResourcesByPurpose.flatMap(_.deployedResources).headOption match {
+        case Some(lzResource) =>
+          F.pure(
+            com.azure.core.management.Region
+              .fromName(lzResource.region)
```

Comment:
Good to know the LZ API response already includes region -- I wasn't sure. 👍

Reply:
it's returning a collection of resources, and each resource has region associated with it...it might make sense to make it so that region is at a higher level in LZ api